### ISSUE TITLE
Total Updates in hiv_pre.sh

### ISSUE THREAD

**Issue by adamoyoung** [microsoft/Graphormer#94]
This question is inspired by #90 

Can I ask what the motivation is for choosing `tot_updates=$((33000*epoch/batch_size/n_gpu))` in `examples/property_prediction/hiv_pre.sh`? 

If I understand correctly, `epoch/batch_size/n_gpu` is the total number of updates that the model will have in this script. If you multiply this by 33000, you are effectively preventing the learning rate from being decayed to `end-learning-rate` by the end of training.

Why wouldn't you simply account for this in the initial settings of `lr` and `end-learning-rate` (i.e. make `end-learning-rate` larger)?

Furthermore, if `warmup_updates=$((tot_updates/10))` doesn't this mean that the entire duration of the script will be warming up (since `3300*epoch/batch_size/n_gpu` is greater than the total number of updates in the script)?

**Comment by adamoyoung** [comment]
I think I misunderstood. There are 32,901 (roughly 32k) compounds in the `molhiv` training set from OGB.

The number of updates per epoch of the `molhiv` training set is actually `33000/batch_size/n_gpu`. Thus, the total number of updates after `epochs` epochs of fine-tuning is `epochs*(33000/batch_size/n_gpu)`.
